### PR TITLE
Feature/bbb 1 sign in

### PR DESCRIPTION
**Pull request by mgilgar**

### CODE REVIEW

File: src/main/java/uk/gov/dft/bluebadge/webapp/la/service/UserServiceImpl.java [hunk 1]
```diff
@@ -36,7 +36,7 @@ public int delete(Long id) {
 
   @Override
   public boolean isAuthorised(String email, String password) {
-    if (email.startsWith("mg")) {
+    if (email.startsWith("accessDenied")) {
       return false;
     }
     return true;
```
Comment (pauljroberts) [inline src/main/java/uk/gov/dft/bluebadge/webapp/la/service/UserServiceImpl.java]: Can this method be deleted?

File: src/test/java/uk/gov/dft/bluebadge/webapp/la/StandaloneMvcTestViewResolver.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+import org.springframework.web.servlet.view.AbstractUrlBasedView;
+import org.springframework.web.servlet.view.InternalResourceView;
+import org.springframework.web.servlet.view.InternalResourceViewResolver;
+
+public class StandaloneMvcTestViewResolver extends InternalResourceViewResolver {
+
+  public StandaloneMvcTestViewResolver() {
+    super();
+  }
```
Comment (pauljroberts) [inline src/test/java/uk/gov/dft/bluebadge/webapp/la/StandaloneMvcTestViewResolver.java]: Does this constructor achieve anything?

Comment (mgilgar) [reply src/test/java/uk/gov/dft/bluebadge/webapp/la/StandaloneMvcTestViewResolver.java]: fixed

File: src/main/java/uk/gov/dft/bluebadge/webapp/la/service/UserServiceImpl.java (hunk 1 above)
Comment (mgilgar) [reply src/main/java/uk/gov/dft/bluebadge/webapp/la/service/UserServiceImpl.java]: fixed

**Comment by mgilgar** [comment]
@pauljroberts  Comments have been addressed

**Comment by mgilgar** [comment]
After being reviewed by @pauljroberts  (for back-end changes) and by @charge-valtech (for front-end changes) and being tested and demoed by @sampathmahavithana I merge this PR to develop.

**Pull request merged**